src/domain/dataverse/port.ts

```diff
@@ -17,4 +17,5 @@ export type DataversePort = {
     offset: number,
     filters: RetrieveDataverseQueryFilters
   ) => TE.TaskEither<Error, RetrieveDataverseResult>
+  abortRequest: () => void
```

Comment:
This naming is a bit confusing because you express in the `port` how the dataverse must be retrieved, (by aborting a **Request**)

IMHO, the `port` only reflects the contract between the logic (hexagon) and the outside (infra), so you should not couple it with the inner implementation of the `gateway`.

To illustrate this, will the name `abortRequest` be appropriated if the `gateway` only parses a local file to retrieve the `dataverse`? No, because there is no request in this case. You'll probably want to cancel the readable stream.

So, I'll suggest to only express here the notion of cancelling the retrieval of the `dataverse`, that is the only purpose of the `port`.